project-generated-code-backend/Backend/Model/Hospital/Floor.cs

```diff
@@ -1,13 +1,11 @@
 ﻿using Backend.Model.Util;
-using Model.Hospital;
-using System.Collections.Generic;
 
 namespace health_clinic_class_diagram.Backend.Model.Hospital
 {
     public class Floor : Entity
     {
         public string Id { get; }
         public string Name { get; set; }
-        public List<Room> Rooms { get; set; }
+        public string BuildingId { get; set; }
```

Comment:
Just check if this is what it's supposed to be, I don't know the details 😄

Reply:
Floor doesn't contain any rooms now. Yes, we want building id in floor.  :))